docs/cicd.md

```diff
@@ -26,6 +26,7 @@ Several workflows exist for different purposes:
 | native_production_delivery | script                 | :heavy_check_mark: | production      | :x:          | :heavy_check_mark: | :x:                |
 | web_beta_delivery          | script                 | :heavy_check_mark: | :x:             | beta         | :heavy_check_mark: | :x:                |
 | web_production_delivery    | script                 | :heavy_check_mark: | :x:             | production   | :heavy_check_mark: | :x:                |
+| e2e_tests                  | commits on main        | :x:                | :x:             | :x:          | :x:                | :x:                |
```

Comment:
Did you test triggering this manually? The command would be like this, right?
`cd tools && yarn trigger-pipeline trigger e2e_tests --api-token <api-token>`?

Reply:
Yes I triggered it manually, see here. I did it by just calling the circleci api, as there is no need for a script, but as Steffen also wishes not to use the api, but have a script, I will add one...

See workflow here:
https://app.circleci.com/pipelines/github/digitalfabrik/integreat-app/8704/workflows/ba0944f6-b798-4c21-9dcc-99ac2ce0a0d9